lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module

```diff
@@ -519,19 +519,45 @@ function dosomething_campaign_get_campaigns_doing($uid = NULL) {
   }
 
   // Get all signups.
-  $signup_nids = dosomething_signup_get_signup_nids_by_uid($uid);
```

Comment:
If we're removing the old functionality do we still want to check for the feature flag?